### PR TITLE
[53776] Android + shared code: updated the todays numbers page with new elements, correct color etc.

### PR DESCRIPTION
**Pull request by Michael-Olesen**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml [hunk 1]
```diff
@@ -65,8 +33,47 @@
...
                 android:layout_gravity="center_horizontal"
                 android:layout_marginHorizontal="24dp"
+                android:paddingBottom="@dimen/defaultVerticalMargin"
                 android:orientation="vertical">
 
+            <LinearLayout
+                    android:id="@+id/daily_numbers_linear_layout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginVertical="21dp"
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml:43]: use default margins

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml [hunk 2]
```diff
@@ -65,8 +33,47 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="center_horizontal"
                 android:layout_marginHorizontal="24dp"
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml:35]: use default margin - and if we don't have one, then create one

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml [hunk 3]
```diff
@@ -92,8 +99,7 @@
                             android:background="@drawable/counter_background_daily_numbers_top"
                             android:paddingHorizontal="8dp"
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml:100]: create default margin to be used

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml [hunk 4]
```diff
@@ -92,8 +99,7 @@
...
                             android:paddingTop="8dp"
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml:101]: create default margin to be used

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml [hunk 5]
```diff
@@ -75,7 +82,7 @@
                     android:paddingVertical="8dp"
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml:82]: create default margin to be used

File: NDB.Covid19/NDB.Covid19/ViewModels/DailyNumbersViewModel.cs [hunk 6]
```diff
@@ -44,6 +58,11 @@ static DailyNumbersViewModel()
             ? string.Format(DAILY_NUMBERS_SUBHEADER, $"{DateUtils.GetDateFromDateTime(LastUpdateFHINumbersDateTime, "m")}", $"{DateUtils.GetDateFromDateTime(LastUpdateFHINumbersDateTime, "t")}")
             : "";
 
+        //For now I assume that FHI will provide vaccination numbers and that these will arrive at the same time as other FHI provided numbers
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/ViewModels/DailyNumbersViewModel.cs:61]: delete this comment

**Review by Frederik-S-N-dk — CHANGES_REQUESTED** [review]

**Comment by Michael-Olesen** [comment]
forgot to add screenshot
todays numbers 1
todays bumbers 2

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml [hunk 7]
```diff
@@ -294,147 +289,313 @@
...
+                        android:layout_height="wrap_content">
+
+                    <TextView
+                            android:id="@+id/daily_numbers_smittestopp_downloads_header_text"
+                            style="@style/DailyNumbersItemHeaderText"
+                            android:layout_weight="0.5"
+                            android:layout_width="0dp"
+                            android:layout_height="match_parent"
+                            android:layout_column="0"
+                            android:layout_marginEnd="16dp"
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml:520]: I see that you are using this "...marginEND = 16 dp" a couple of places. Please create a defualt dimen for this

**Review by Frederik-S-N-dk — CHANGES_REQUESTED** [review]

**Review by Frederik-S-N-dk — APPROVED** [review]

**Pull request merged**